frontend/src/features/workspace/components/DuplicateFormModal/DupeFormWizardProvider.tsx

```diff
@@ -14,17 +14,18 @@ import {
   CreateFormWizardContextReturn,
 } from '../CreateFormModal/CreateFormWizardContext'
 import { useCommonFormWizardProvider } from '../CreateFormModal/CreateFormWizardProvider'
-import { useWorkspaceRowsContext } from '../WorkspaceFormRow/WorkspaceRowsContext'
 
-export const useDupeFormWizardContext = (): CreateFormWizardContextReturn => {
+export const useDupeFormWizardContext = (
+  formId: string,
+  isTemplate: boolean,
+): CreateFormWizardContextReturn => {
   const { data: dashboardForms, isLoading: isWorkspaceLoading } = useWorkspace()
-  const { activeFormMeta } = useWorkspaceRowsContext()
 
   const { data: previewFormData, isLoading: isPreviewFormLoading } =
     usePreviewForm(
-      activeFormMeta?._id ?? '',
+      formId,
       // Stop querying once submissionData is present.
-      /* enabled= */ !!activeFormMeta,
+      /* enabled= */ !!formId,
```

Comment:
because of this check, i wonder if actually we actually need `formId` to be provided. could we possibly make `formId` an optional argument, then in `WorkspaceRowsContext.tsx` we don't have to handle the undefined case?

```javascript
        formId={activeFormMeta?._id}

Reply:
i left it as is as it's technically needed for in the queries `usePreviewForm(formId)` / `useTemplateForm(formId)`

Reply:
haha as in, we can leave it undefined, then in the dupe form wizard provider do

```javascript
const useDupeFormWizardContext = (formId = '', isTemplate) => ...